src/utility/LottieOptions.tsx

```diff
@@ -6,13 +6,22 @@ import projectManagementLottie from "../images/lottie/projectManagement_lottie.j
 import projectTeamLottie from "../images/lottie/projectTeamLottie.json";
 import handshakeLottie from "../images/lottie/handshake_lottie.json";
 import teamproject from "../images/lottie/teamproject.json";
+import applyPageTeamLottie from "../images/lottie/applyPageTeamLottie.json";
 
+export const applyPageTeamLottieOptions = {
+  loop: true,
+  autoplay: true,
+  animationData: applyPageTeamLottie,
+  rendererSettings: {
+    preserveAspectRatio: "xMidYMid slice",
+  },
+};
 export const projectTeamLottieOptions = {
   loop: true,
   autoplay: true,
   animationData: projectTeamLottie,
   rendererSettings: {
-    preserveAspectRatio: "xMidYMid slice",
+    preserveAspectRatio: "xMidYMin slice",
```

Comment:
Why did this change?

Reply:
I was testing the preserveAspectRatio to see its effect, so I deleted and then added the line back, but that's the original line, I reverted all the changes so nothing should change I believe